src/components/hint/index.tsx

```diff
@@ -549,40 +546,24 @@ class Hint extends Component<HintProps, HintState> {
   }
 
   render() {
-    const {onBackgroundPress, backdropColor, testID} = this.props;
-
     if (!this.props.visible && this.state.hintUnmounted) {
       return this.props.children || null;
     }
 
     return (
       <>
         {this.renderChildren()}
```

Comment:
I suggest keeping the old implementation commented, till we know for sure there are no issues. 
Maybe even support both with a new prop (`useModal` for example)

Reply:
Done